packages/back-end/src/models/FeatureModel.ts

```diff
@@ -48,6 +53,12 @@ const featureSchema = new mongoose.Schema({
         },
       ],
       namespace: {},
+      scheduleRules: [
+        {
+          timestamp: String,
```

Comment:
I've gone back and forth on this, but I'm not sure I have enough context to know which is the right direction.

Initially I thought about adding it so we can change the shape of the `nextScheduledUpdate` property from a single date, to a `ScheduleRule` object. The idea being it could make getting data about the next scheduled update a bit easier. Currently, we have to loop through the array of rules and find the first rule where the timestamp is greater than the currentDate.

If we added the _id (or, just an `id` property, we could instead get the scheduleRule.id from the `nextScheduledUpdate` and then access that index in the scheduleRules array. 

I left it out given that it could increase complexity of the cron job's query. But open to discussion on if other's think that is a good enough reason to not add the `_id` or just an `id` property.

Reply:
Sure, if we are targeting these ScheduleRules by id somewhere (cron, etc), then it does make sense to keep the field I think.